department-of-veterans-affairs/caseflow
# APPEALS-45336

Resolves [APPEALS-45336 Creation of Saved Searches page and ‘View saved searches’ button on Generate task report page](https://jira.devops.va.gov/browse/APPEALS-45336)

# Description
Creates a blank Saved Searches page and button to reach it from the reports page

## Acceptance Criteria
- [ ] Creation of ‘View saved searches’ button on Generate task report page 
    - [ ] On click, routes to new Saved Searches page (/report/searches) 
- [ ] Creation of Saved Searches page 
    - [ ] Page should only be accessible by VHA Admins 
    - [ ] Creation of links/button: 
        - [ ] < Back to Generate task report 


## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Log in as VHAADMIN
3. Go to http://localhost:3000/decision_reviews/vha/report and verify the presence of View saved searches
4. Click View saved searches and verify the new page and back link


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
